README.md

```diff
@@ -1,78 +1,100 @@
+* **Bundler:** Install it with `gem install bundler`.
+* **Database:** Ensure PostgreSQL server is running and `libpq-dev` (or equivalent) is installed for native gem compilation.
+* **Yarn:** Install it using `npm install -g yarn`.
 
-And then precompile the assets 
-`rake assets:precompile`
+## Setup Instructions
 
-Create the database with `rake db:setup db:migrate`
+1.  **Clone the Repository**
 
-Then run `rails server`
+    Set up environment variables 
+    ```bash
+    cp .env.example .env
+    ```
 
-## AZURE Blob storage ##
+2.  **Install Dependencies:**
 
-Until we refactor file uploader to use active storage we have to use storage for Demo or AAT env.
-Example for AAT(Staging)
+    ```bash
+    bundle install
+    ```
+    You will need to install the GovUK FrontEnd package
+    https://frontend.design-system.service.gov.uk/installing-with-npm/#requirements
 
-### Add your IP to storoage ###
-Add your IP to whitelisting on relevant storage account
-Storage account -> Security + networking -> Networking -> Firewall and virtual networks tab
-https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/58a2ce36-4e09-467b-8330-d164aa559c68/resourceGroups/tt_stg_taxtribunalsazure_resource_group/providers/Microsoft.Storage/storageAccounts/stgttfilestore/networking
+    ```bash
+    npm install govuk-frontend --save
+    ```
 
-Firewall section -> Add IP ranges to allow access from the internet or your on-premises networks.
-There should be checkbox with your IP address. Check it and click on save (at the top of the page)
+    And then precompile the assets
 
-### Load storoage credentials on your local ###
-Find a tax-tribunals-cft-aat key/vault on portal (https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourceGroups/tax-tribunals-aat/providers/Microsoft.KeyVault/vaults/tax-tribunals-cft-aat/overview)
+    ```bash
+    rake assets:precompile
+    ```
+3.  **Configure Database:**
 
-Copy azure-storage-account, azure-storage-container and azure-storage-key into your localhost .env setup accordingly.
+    * Edit `config/database.yml` and configure your database settings. Replace the placeholders with your database credentials.
 
+    * Create the database:
 
- 
-## Mutation testing performance issues and resolution
+        ```bash
+        rails db:create
+        ```
 
-### Background
+    * Run Migrations:
+        ```bash
+        rails db:migrate
+        ```
 
-At the end of March 2017, we encountered problems with CircleCI and
-mutation testing.  Calls to `self` in `TribunalCase` were causing heavy
-recursion in two of the mutation tests run against that model's spec [1].
-These consumed large amounts of memory, ran very slowly, and effectively
-blocked two mutants threads for hours on a locally running instance.
+4.  **Install JavaScript Dependencies:**
 
-### CircleCI vs. Travis
```

Comment:
I think you should remove the Travis think and left a comment for Jenkins CI and Jenkins CNP file.